packages/BetterPhpDocParser/PhpDocParser/ClassAnnotationMatcher.php

```diff
@@ -31,7 +31,11 @@ public function __construct(
     ) {
     }
 
-    public function resolveTagFullyQualifiedName(string $tag, Node $node): string
+    public function resolveTagFullyQualifiedName(
+        string $tag,
+        Node $node,
+        bool $returnNullOnUnknownClass = false
```

Comment:
Alternatively to a third parameter, we could add a new method `resolveTagFullyQualifiedNameWithNullIfUnknown` (naming is TBD)..